projects/ngx-datatable/src/lib/components/datatable.component.ts

```diff
@@ -265,6 +265,12 @@ export class DatatableComponent implements OnInit, DoCheck, AfterViewInit, After
    */
   @Input() loadingIndicator = false;
 
+  /**
+   * Show ghosts loader on each cell.
```

Comment:
Typo I guess -> the `ghost loader` as a whole is plural.

```suggestion
   * Show ghost loaders on each cell.
```